README.md

```diff
@@ -25,28 +35,27 @@ Azure service crates generated from [Azure REST API Specifications](https://gith
 
 🚨 WARNING 🚨: This project is under active development. Be aware that large breaking changes will happen before 1.0 is reached.
 
+### Historical releases
+
 This project is the successor to the `azure_sdk*` crates from [MindFlavor/AzureSDKForRust](https://github.com/MindFlavor/AzureSDKForRust). The crates have been renamed, so those older crates should be considered fully deprecated. See [history](HISTORY.md) for more details.
 
-## Project Structure
+## Reporting security issues and security bugs
+
+Security issues and bugs should be reported privately, via email, to the Microsoft Security Response Center (MSRC) <secure@microsoft.com>. You should receive a response within 24 hours. If for some reason you do not, please follow up via email to ensure we received your original message. Further information, including the MSRC PGP key, can be found in the [Security TechCenter](https://www.microsoft.com/msrc/faqs-report-an-issue).
 
-Each supported Azure service is its own separate crate.
+## Need help?
 
-Building each crate should be as straight forward as `cargo build`, but check each crate's README for more specific information.
+* File an issue via [Github Issues](https://github.com/Azure/azure-sdk-for-rust/issues)
+* Check [previous questions](https://stackoverflow.com/questions/tagged/azure+rust) or ask new ones on StackOverflow using `azure` and `rust` tags.
 
-### Mock testing framework
+## Community
 
-This library comes with a testing framework that executes against prerecorded sessions to quickly validate code changes without incurring in Azure costs. You can read more about it in the [Mock testing framework's README](https://github.com/Azure/azure-sdk-for-rust/tree/feature/track2/doc/mock_transport.md).
+## Contribute
```

Comment:
Empty heading.

We should put information that was here in a `CONTRIBUTING.md` at some point.